api/common/interfaces.go

```diff
@@ -668,6 +669,37 @@ func getSubscription(s Serverer, params map[string]interface{}) map[string]inter
 	})
 }
 
+func getRegName(s Serverer, params map[string]interface{}) map[string]interface{} {
+	if len(params) < 3 {
+		return respPacking(INVALID_PARAMS, "length of params is less than 3")
+	}
+
+	name, ok := params["name"].(string)
+	if !ok {
+		return respPacking(INVALID_PARAMS, "name should be a string")
+	}
+
+	registrant, ok := params["registrant"].(string)
+	if !ok {
+		return respPacking(INVALID_PARAMS, "registrant should be a string")
+	}
+
+	expiresAt, ok := params["expiresAt"].(string)
+	if !ok {
```

Comment:
For better convenient of API, `expiresAt` should be an option, not MUST params.

Reply:
done